net-im/synapse/synapse-1.95.0-r1.ebuild

```diff
@@ -121,7 +122,7 @@ RDEPEND="
 	dev-python/signedjson[${PYTHON_USEDEP}]
 	dev-python/sortedcontainers[${PYTHON_USEDEP}]
 	dev-python/treq[${PYTHON_USEDEP}]
-	dev-python/twisted[${PYTHON_USEDEP}]
+	<=dev-python/twisted-23.8.0[${PYTHON_USEDEP}]
```

Comment:
Careful - this will exclude -rN which likely isn't intentional.

Reply:
I see, thanks! I have changed it to `<dev-python/twisted-23.8.1` in recent FP.